brainscore_vision/models/effnetb1_272x240/model.py

```diff
@@ -0,0 +1,131 @@
+from PIL import Image
+import numpy as np
+import timm
+from timm.data import resolve_data_config
+from timm.data.transforms_factory import create_transform
+import torch.nn as nn
+from albumentations import (
+    Compose, Normalize, Resize,CenterCrop
+    )
+from albumentations.pytorch import ToTensorV2
+# This is an example implementation for submitting alexnet as a pytorch model
+# If you use pytorch, don't forget to add it to the setup.py
+
+# Attention: It is important, that the wrapper identifier is unique per model!
+# The results will otherwise be the same due to brain-scores internal result caching mechanism.
+# Please load your pytorch model for usage in CPU. There won't be GPUs available for scoring your model.
+# If the model requires a GPU, contact the brain-score team directly.
+from brainscore_vision.model_helpers.check_submission import check_models
+
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+import os 
+
+image_resize = 272
+image_crop = 240
+norm_mean = [0.485, 0.456, 0.406] 
+norm_std = [0.229, 0.224, 0.225]
+
+def custom_image_preprocess(images, **kwargs):
+    
+    transforms_val = Compose([
+        Resize(image_resize, image_resize),
+        CenterCrop(image_crop, image_crop),
+        Normalize(mean=norm_mean,std=norm_std,),
+        ToTensorV2()])
+    
+    images = [np.array(pillow_image) for pillow_image in images]
+    images = [transforms_val(image=image)["image"] for image in images]
+    images = np.stack(images)
+
+    return images
+
+def load_preprocess_images_custom(image_filepaths, preprocess_images=custom_image_preprocess,  **kwargs):
+    images = [load_image(image_filepath) for image_filepath in image_filepaths]
+    images = preprocess_images(images, **kwargs)
+    return images
+
+def load_image(image_filepath):
+    with Image.open(image_filepath) as pil_image:
+        if 'L' not in pil_image.mode.upper() and 'A' not in pil_image.mode.upper()\
+                and 'P' not in pil_image.mode.upper():  # not binary and not alpha and not palletized
+            # work around to https://github.com/python-pillow/Pillow/issues/1144,
+            # see https://stackoverflow.com/a/30376272/2225200
+            return pil_image.copy()
+        else:  # make sure potential binary images are in RGB
+            rgb_image = Image.new("RGB", pil_image.size)
+            rgb_image.paste(pil_image)
+            return rgb_image
+
+def get_model_list():
+    return ['effnetb1_272x240']
+
+
+class EffNetBX(nn.Module):
+    def __init__(self,):
+        super().__init__ ()
+        self.efnet_model = timm.create_model('tf_efficientnet_b1_ns', pretrained=True)
+
+    def forward(self, x):
+        x = self.efnet_model(x)
+        return x
+
+def get_model(name):
+    assert name == 'effnetb1_272x240'
+    model_tf_efficientnet_b1_ns= EffNetBX()
+    dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
These weights should be hosted on S3

Reply:
It doesn't even look like this model is pulling in any new weights. Am I mistaken here?